iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/BusBreakerVoltageLevel.java

```diff
@@ -41,6 +42,13 @@
  */
 class BusBreakerVoltageLevel extends AbstractVoltageLevel {
 
+    private static final int DEFAULT_NODE_LIMIT = 1000;
+
+    public static final int NODE_LIMIT = PlatformConfig.defaultConfig()
+            .getOptionalModuleConfig("iidm")
+            .map(moduleConfig -> moduleConfig.getIntProperty("node-limit", DEFAULT_NODE_LIMIT))
+            .orElse(DEFAULT_NODE_LIMIT);
```

Comment:
@flo-dup I tried to copy the example you provided in #1845, but not totally sure if this is correct.

Reply:
seems good to me!

Reply:
seems good to me!